package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "watch": "webpack --watch",
+    "build": "webpack"
+  },
```

Comment:
- [OPTIONAL] There is no way to start the project in development mode. When I run `npm start`, I get the following error:

![image](https://github.com/Zohra-Neda/Leaderboard/assets/77106962/1db9e3b6-1e89-4bc3-bf75-c6279dfeeb07)

To fix this, you can include the start command to the missing `"scripts"` by doing this 😎 💻:

```JavaScript
"start": "webpack serve --open"
```